fast-referencedata-service/proto/referencedata_service.proto

```
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package gpp.g3.referencedata;
```

Buf (1.54.0)

Files with package "gpp.g3.referencedata" must be within a directory "gpp/g3/referencedata" relative to root but were in directory "fast-referencedata-service/proto".

(PACKAGE_DIRECTORY_MATCH)